reference/implementing-a-concept-exercise.md

```diff
@@ -67,4 +66,4 @@ See [documentation files][docs-concept-exercises-documentation-files].
 [reference]: https://github.com/exercism/v3/blob/main/reference
 [docs-concept-exercises]: https://github.com/exercism/v3/blob/main/docs/concept-exercises.md
 [docs-concept-exercises-documentation-files]: https://github.com/exercism/v3/blob/main/docs/concept-exercises.md#documentation-files
-[anatomy-of-a-concept-exercise]: https://www.youtube.com/watch?v=gkbBqd7hPrA
+[anatomy-of-a-concept-exercise]: https://www.youtube.com/watch?v=gkbBqd7hPrA
```

Comment:
The newline at the end of this file should not be deleted. Please put it back.

Reply:
(reason: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)